opensearchpy/_async/transport.py

```diff
@@ -54,6 +54,7 @@ class AsyncTransport(Transport):
     """
 
     DEFAULT_CONNECTION_CLASS = AIOHttpConnection
+    DEFAULT_HOSTS_PARENT_CLASS_ARG = []
```

Comment:
This is unused, remove?

Reply:
@dblock You are correct, it is not used in `opensearchpy/_async/transport.py` and not present in main branch